src/pages/_app.js

```diff
@@ -313,72 +313,70 @@ export default class App extends BaseApp {
     };
 
     return (
-      <Container>
```

Comment:
Next.js 9 doesn't need this component wrapper anymore. Everything below this line in this file is just Prettier's re-indentation, feel free to ignore :)